Examples/test-suite/common.mk

```diff
@@ -706,8 +706,9 @@ CPP20_TEST_BROKEN = \
 # Doxygen comment translation (currently Python and Java) and only if not
```

Comment:
This list of languages needs updating (or perhaps better just removing as it just duplicates the information that's in the `python_HAS_DOXYGEN := 1`, etc lines right below).

Reply:
Yes, sorted now.